typescript/commands/build.py

```diff
@@ -0,0 +1,40 @@
+import sublime_plugin
+import sublime
+import os
+from ..libs.global_vars import IS_ST2
+
+
+class TypescriptBuildCommand(sublime_plugin.WindowCommand):
+    def run(self):
+        file_name = self.window.active_view().file_name()
+        directory = os.path.dirname(file_name)
+        if "tsconfig.json" in os.listdir(directory):
+            self.window.run_command("exec", {
+                "cmd": ["tsc"],
```

Comment:
I thought we were going to use the TypeScript compiler that effectively ships with the Sublime plugin (i.e. is included in tsserver.js).  Otherwise if you have a different version of the TypeScript Node package installed than the plug-in version, you might get different errors/features from your build than you do from the language service while editing - which would be weird.

As we're already locating Node and tsserver.js to spin up the plugin service, it shouldn't be too hard to use the same located files to run the compilation for the build.  What do you think?